### PR TITLE
Prepare to publish

### PR DESCRIPTION
**Pull request by aspacca**
Cleanup/Docs
## What does this PR do?
Remove IAM policies from SAR template and add documentation for the user to provide autonomously

Refactor secrets manager expander

## Why is it important?
Previous SAR template required too broad access to AWS resources, that would not fit security requirements

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have made corresponding change to the default configuration files
- [x] I have added tests that prove my fix is effective or that my feature works
~- [ ] I have added an entry in `CHANGELOG.md`~

## Author's Checklist

- [ ]

## Related issues

-

### CODE REVIEW

**Comment by aspacca** [comment]
@bogdan-copocean please check the commit related to secrets manager expander

@endorama please, give an overall review

**Review by bogdan-copocean — APPROVED** [review]
good to go :)

**Review by endorama — APPROVED** [review]

File: docs/README-AWS.md [hunk 1]
```diff
@@ -64,57 +85,153 @@ Resources:
     Type: AWS::Serverless::Application
     Properties:
       Location:
-        ApplicationId: 'arn:aws:serverlessrepo:%REGION%:%ELASTIC_ACCOUNT_ID%:applications/elastic-serverless-forwarder'
+        ApplicationId: 'arn:aws:serverlessrepo:%REGION%:267093732750:applications/elastic-serverless-forwarder'
```
Comment (kaiyan-sheng) [inline docs/README-AWS.md:88]: Is this intentional?

Comment (aspacca) [reply docs/README-AWS.md:88]: yes, it's the AWS account where it will be published

**Pull request merged**